impl_core/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -119,6 +125,23 @@ private CumulativeMutableViewData(View view, Timestamp start, MetricMap metricMa
       }
     }
 
+    @Override
+    Metric toMetric(Timestamp now, StatsCollectionState state) {
+      if (state == StatsCollectionState.DISABLED) {
+        return null;
+      }
+      MetricDescriptor.Type type = metricDescriptor.getType();
+      List<TimeSeriesCumulative> timeSeriesCumulatives = new ArrayList<TimeSeriesCumulative>();
```

Comment:
This list should be created based on `type`. If `type` is `GAUGE_DOUBLE` or `GAUGE_INT64`, this should be `TimeSeriesGauge`. See https://github.com/census-instrumentation/opencensus-java/pull/1368/files#diff-05a76803285c4ebb76d41d8822baacbfL117.

Reply:
Done.